tonic-examples/src/tls_client_auth/client.rs

```diff
@@ -13,7 +13,7 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     let client_key = tokio::fs::read("tonic-examples/data/tls/client1.key").await?;
     let client_identity = Identity::from_pem(client_cert, client_key);
 
-    let tls = ClientTlsConfig::with_openssl()
+    let tls = ClientTlsConfig::with_rustls()
```

Comment:
Was this supposed to change in this PR?

Reply:
Yes, but I think it was missed in another PR so I thought i might as well change this now. I want all examples with rustls :)

We really only need openssl for interop since the certs that grpc-go uses are too small for the min that rustls supports.